gamemode/modules/base/sv_util.lua

```diff
@@ -10,7 +10,7 @@ function DarkRP.notify(ply, msgtype, len, msg)
     end
 
     local rcp = RecipientFilter()
-    for _, v in pairs(ply) do
+    for _, v in ipairs(ply) do
```

Comment:
Same, because this is an interface function, we cannot assume every caller calls it nicely. Changing to ipairs could break addons.